run.py

```diff
@@ -457,15 +463,3 @@ def run_state_level_forecast(
 
     pool.close()
     pool.join()
-
-
-if __name__ == "__main__":
-    logging.basicConfig(level=logging.INFO)
-    with public_data_hash(os.getenv('COVID_DATA_PUBLIC_HASH', None)) as git_hash:
-        # @TODO: Record git hash in output data for reproducibility
-        # @TODO: Remove interventions override once support is in the Harvard model.
-        min_date = datetime.datetime(2020, 3, 7)
-        max_date = datetime.datetime(2020, 7, 6)
-        # build_county_summary()
-        run_county_level_forecast(min_date, max_date)
-        run_state_level_forecast(min_date, max_date)
```

Comment:
So you deleted this (again).  I don't know if that is okay.  Is this going to break other people's workflow?  And why does run.py exist if it doesn't have a main entry point?  

If run.py is deprecated / whatever, maybe you should make it print a message saying, "run.py is deprecated.  Run this instead: ..." or something?

Reply:
Now that we have a consistent way of running the deploys (./deploy.sh) it's easier to point people away from this, so i feel better about it now. 

I'm down with the deprecation method print message though.

run.py always did a lot of things, in the plan we have, those different tasks have new homes.  If it doesn't exist, I can make a ticket to split run.py up.

Reply:
Warning added!